packages/connect-react/examples/nextjs/src/app/page.tsx

```
@@ -21,6 +22,11 @@ export default function Home() {
     text: "hello slack!",
   });
 
+  const [
+    dynamicProps,
+    setDynamicProps,
+  ] = useState<DynamicProps<{}>>();
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)